cluster/gce/config-default.sh

```diff
@@ -285,6 +285,7 @@ NODE_PROBLEM_DETECTOR_TAR_HASH="${NODE_PROBLEM_DETECTOR_TAR_HASH:-}"
 NODE_PROBLEM_DETECTOR_RELEASE_PATH="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-}"
 NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS="${NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS:-}"
 
+CNI_STORAGE_PATH="${CNI_STORAGE_PATH:-kubernetes-release/network-plugins}"
```

Comment:
do we want this relative path or absolute path?

Reply:
why not the full URL

Reply:
Because this is using google storage, I am not seeing a reason to use a none-google storage. 
https://storage.googleapis.com/${CNI_STORAGE_PATH}/${cni_tar}

Reply:
For testing? we also don't have to change any code in the future if this assumption is no longer true...

Reply:
Make sense, updated. Thank you for reviewing.